core/test/utils/api.js

```diff
@@ -126,7 +46,7 @@ function checkResponseValue(jsonResponse, expectedProperties) {
 function checkResponse(jsonResponse, objectType, additionalProperties, missingProperties, onlyProperties, options) {
     options = options || {};
 
-    var checkProperties = options.public ? (expectedProperties[objectType].public || expectedProperties[objectType]) : (expectedProperties[objectType].default || expectedProperties[objectType]);
+    let checkProperties = options.public ? (this.expectedProperties[objectType].public || this.expectedProperties[objectType]) : (this.expectedProperties[objectType].default || this.expectedProperties[objectType]);
```

Comment:
This commit is big, because of all the replacements.
But only this piece is important. Each API version can define it's own expected props object.